ark/utils/notebooks_test.py

```diff
@@ -23,3 +32,93 @@ def test_example_spatial_analysis():
 
 def test_example_neighborhood_analysis():
     _exec_notebook('example_neighborhood_analysis_script.ipynb')
+
+
+# testing specific inputs for Segment_Image_Data
+# test mibitiff, 6000 seconds = default timeout on Travis
+@testbook(SEGMENT_IMAGE_DATA_PATH, timeout=6000)
+def test_segment_image_data_mibitiff(tb):
+    # create input files, set separate names for mibitiffs to avoid confusion
+    notebooks_test_utils.segment_notebook_setup(tb,
+                                                deepcell_tiff_dir="test_mibitiff",
+                                                deepcell_input_dir="test_mibitiff_input",
+                                                deepcell_output_dir="test_mibitiff_output",
+                                                single_cell_dir="test_mibitiff_single_cell",
+                                                is_mibitiff=True)
+
+    # default fov setting, standard nucs/mems setting
+    notebooks_test_utils.fov_channel_input_set(tb,
+                                               nucs_list=['chan0'],
+                                               mems_list=['chan1', 'chan2'])
+
+    # default fov setting, nucs set to None
+    notebooks_test_utils.fov_channel_input_set(tb,
```

Comment:
I don't think this is necessary. We already test `create_deepcell_input` with all permutations of blank and non-blank channel inputs, right? Or is this covering something different?

Reply:
You're right, we already test that. I'll remove those tests and just leave the one where we hard code a `fovs` list in.